arrow-fx/src/main/kotlin/arrow/fx/Resource.kt

```diff
@@ -165,57 +168,109 @@ interface Resource<F, E, A> : ResourceOf<F, E, A> {
    *
    * fun main() {
    *   //sampleStart
-   *   val program = Resource(::acquireResource, ::releaseResource, IO.bracket()).invoke {
+   *   val program = Resource(::acquireResource, ::releaseResource, IO.bracket()).use {
    *     IO { println("Expensive resource under use! $it") }
    *   }
    *   //sampleEnd
    *   program.fix().unsafeRunSync()
    * }
    * ```
    */
-  operator fun <C> invoke(use: (A) -> Kind<F, C>): Kind<F, C>
```

Comment:
This function should be `@Deprecated` in `0.10.x` and removed in `0.11.0`.
`Resource` is already available in the latest stable version `0.10.4`